recipes/ray/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+ln -sf $PREFIX/lib/RayPlatform RayPlatform
+make
+cp -R Ray $PREFIX/bin/
+find scripts -type f \( -name "*.sh" -o -name "*.py" \) -exec cp {} $PREFIX/bin/ \;
```

Comment:
If `ray` is a pile of scripts (with some ancillary files), consider copying everything to a package-specific directory within `$PREFIX/share` or `$PREFIX/opt`, similar to [this recipe](https://github.com/bioconda/bioconda-recipes/blob/master/recipes/trimmomatic/build.sh) for trimmomatic. Doing so is more in line with the [Linux filesystem hierarchy](https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard), and it helps avoid collisions. You can symlink scripts to `$PREFIX/bin` from there.